README.md

```diff
@@ -92,7 +92,7 @@ cover each phase in this section and a more detailed version can be found
 2. When a pod is created, API Server mounts the cert/key according to the
    service account using [Kubernetes
    secrets](https://kubernetes.io/docs/concepts/configuration/secret/).
-3. [Istio Manager](https://github.com/istio/manager/blob/master/doc/design.md)
+3. [Istio-Manager](https://github.com/istio/manager/blob/master/doc/design.md)
```

Comment:
This is an interesting naming convention. https://screenshot.googleplex.com/YF1B3pPoq5T, not sure if we should change it for Auth as well. I will let you decide.